acq4/pyqtgraph/widgets/SpinBox.py

```diff
@@ -143,7 +143,7 @@ def setOpts(self, **opts):
         """
         #print opts
         for k in opts:
-            if k == 'bounds':
+            if k == 'bounds' or k == 'range':
```

Comment:
Let's just fix range->bounds wherever we see it. I'd prefer to keep the SpinBox API clean.